test/test.html

```diff
@@ -0,0 +1,23 @@
+<html>
+	<head>
+		<meta charset="utf-8">
+		<!-- encoding must be set for mocha's special characters to render properly -->
```

Comment:
I don't think we need the comment, encoding should always be declared first thing in the `head` tag to avoid some forms of XSS attacks.